library/include/rocrand/rocrand_common.h

```diff
@@ -41,10 +41,6 @@
 
 #define ROCRAND_KERNEL __global__ static
 
-#ifndef FQUALIFIERS
-#define FQUALIFIERS __forceinline__ __device__
```

Comment:
@parbenc Why is FQUALIFIERS being removed?  It seems like a useful and heavily-used macro.

Reply:
The main problem is that it is defined in multiple locations, with different content. Another issue is that it is less readable, than just writing out `__forceinline__ __device__`. In the end we decided, that it is better to just remove it.

Reply:
Okay.